_projects/nextgen_action_system.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "Nextgen action system"
```

Comment:
Use title case ;) --> `Nextgen Action System`
Also, because people don't know that "nextgen" stands for "next.generation", maybe expand this expression ;)